src/psyclone/psyGen.py

```diff
@@ -3526,8 +3412,13 @@ def __init__(self, call, arg_info, access):
             # Use our namespace manager to create a unique name unless
             # the context and label match in which case return the
             # previous name.
-            self._name = self._name_space_manager.create_name(
-                root_name=self._orig_name, context="AlgArgs", label=self._text)
+            # There are unit-tests where we create Arguments without an
```

Comment:
Remove comment about namespace manager above

Reply:
Removed this and other 'namespace' comments I found.